src/utils/time-utils.ts

```diff
@@ -226,3 +228,37 @@ export const getSprintStart = (date: string) => {
 export const getSprintEnd = (date: string) => {
   return getSprintStart(date).plus({ days: 11 });
 };
+
+/**
+ * Calculate color for vacation days from vacation days
+ *
+ * @param user Keycloak user
+ */
+export const getVacationColors = (user: User)=> {
+  let spentVacationsColor = theme.palette.error.main;
+  let unspentVacationsColor = theme.palette.error.main;
+
+  if (user && parseVacationDays(user.attributes?.vacationDaysByYear ?? ["2024:30"])[new Date().getFullYear()] > 0) {
```

Comment:
Maybe we shouldn't hardcode the 2024, but as the current full year e.g. new Date().getFullYear() and insert it into th estring to replace 2024?

Reply:
I also wonder if it might be worth breaking this condition down a little, by declaring a currentYear variable before the condition and then having [currentYear] rather than [new Date().getFullYear()]. Improves readability a tiny bit maybe